### PR TITLE
Set users to highlight on home page

### PR DESCRIPTION
**Pull request by mosabua**
This PR replaces the current setup where each deployment select a random 16 users and displays them on the home page. The new approach allows us to configure which users show on the home page all the time. 

We also considered using a carousel with all users or a subset but found in our research that carousels are now often frowned upon and have usability issues. We will also have a similar list of supported integrations soon so we are either delaying carousel until that is out or completely abandoning that idea .. at least for now.

Asking for code review and also review of selection of users to highlight. Should be a total of 16 to fit the two rows of 8 on the homepage.

Starting with feedback from @colebow

### CODE REVIEW

File: _includes/users-summary.html [hunk 1]
```diff
@@ -1,18 +1,20 @@
 <div class="user-logos-container">
-{%- assign selection = site.data.users | sample: 16 -%}
+{%- assign selection = site.data.users -%}
 {%- for user in selection -%}
```
Comment (colebow) [inline _includes/users-summary.html:3]: Can we not just do `for user in site.data.users`?

**Review by colebow — APPROVED** [review]
LGTM

File: _includes/users-summary.html (hunk 1 above)
Comment (mosabua) [reply _includes/users-summary.html:3]: yes.. forgot about going back to that..

File: _data/users.yml [hunk 2]
```diff
@@ -373,6 +409,7 @@
   urltext:
   url: https://goraft.tech
 - name: Rapido
+  highlight: true
```
Comment (colebow) [inline _data/users.yml:412]: I think JioSaavn or Myntra might be better to highlight over Rapido - all three are comparably successful and India-based, but we've already got a couple of other transportation companies on this list with DiDi and Lyft.

**Review by colebow — APPROVED** [review]
LGTM, I think this is a huge improvement.

File: _data/users.yml (hunk 2 above)
Comment (mosabua) [reply _data/users.yml:412]: Which one? 

Also can you do a PR for udpating the rapido logo please

Comment (mosabua) [reply _data/users.yml]: Changed to stripe after discussing on slack

**Comment by mosabua** [comment]
We also discussed with the maintainers on slack. Getting this live as is now. Will follow up with changes over time.

**Pull request merged**